stanford_events_importer.install

```diff
@@ -760,3 +760,11 @@ function stanford_events_importer_update_7304(&$sandbox) {
     field_cache_clear();
   }
 }
+
+/**
+ * Update variables to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7305(&$sandbox) {
+  variable_set('stanford_events_category_source', 'http://events-legacy.stanford.edu/xml/drupal/v2.php?category-list');
+  variable_set('stanford_events_organization_source', 'http://events-legacy.stanford.edu/xml/drupal/v2.php?organization-list');
```

Comment:
The "correct" way would be the fetch the current variable and swap the domains. But would anyone actually change these variable.. NO. so I don't really care much.

Reply:
Yeah, that's basically my train of thought. We _could_ do a `variable_get()`, and then an `if...else`, but I don't care that much either. I'll deal with the consequences of this decision. `/blame @jbickar`